src/installing-workflow/configuring-object-storage.md

```diff
@@ -32,6 +32,7 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
             ```
             export AWS_ACCESS_KEY, AWS_SECRET_KEY, AWS_REGISTRY_BUCKET, AWS_DATABASE_BUCKET, AWS_BUILDER_BUCKET, S3_REGION
             ```
+              * If you want to use [IAM credentials][aws-iam] you need not set AWS_ACCESS_KEY and AWS_SECRET_KEY.
```

Comment:
The docs aren't (yet) consistent about this in the docs, but it's a good suggestion not to use "you" (or "we" or "I") unless it makes things awkward. Maybe:
`To use [IAM credentials][aws-iam], it is not necessary to set AWS_ACCESS_KEY or AWS_SECRET_KEY.`